commit.go

```diff
@@ -106,6 +106,13 @@ type PushOptions struct {
 	Quiet bool
 }
 
+var (
+	// commitPolicy bypasses any signing requirements when committing containers to images
+	commitPolicy = []byte(`{"default": [ { "type": "insecureAcceptAnything" } ] }`)
+	// pushPolicy bypasses any signing requirements when pushing (copying) images from local storage
+	pushPolicy = commitPolicy
```

Comment:
1. Please use the Go API:
    ```go
    &signature.Policy{Default: []signature.PolicyRequirement{signature.NewPRInsecureAcceptAnything()}}
    ```
    so that the code visibly breaks on a typo, or if the policy format ever changed.

2. Yes, the change seems to be correct right now, but for extra paranoia I’d prefer an explicit
    - default = reject
    - containers-storage = insecureAcceptAnything

    configuration, so that if ever any of the code is refactored or extended to allow non-storage or remote sources, it does not accidentally bypass policy. (In that case, I think hard-failing is sufficient, because it “can’t happen”; if you wanted to be fancy, you could load the actual system policy and only override the containers-storage configuration, but I don’t think it’s warranted.

Reply:
Very convincing on both counts.  Changed it to create an API struct rather than parsing a hard-coded configuration in text format.